### DISCUSSION TITLE
【BUG】v1.88.0 Photo Uploaded by PC cannot correctly display in 【Photo Menu】

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#5335] [Q&A]
![1701099064941](https://github.com/immich-app/immich/assets/151803203/0f6f6a70-ff6e-46ab-a270-ccb73f1faede)
**### please refer to my screeshot，when i upload the photo by using immich web on PC, i cannot see it in Photo Menu, otherwise, it could be found in 【Explore Menu --> Recently added】，very confused for it，i think this phenomena should belong to BUG，appreciate to your advice for this，thanks~！**

**Comment by jrasm91** [comment]
What is the date on the photo?

**Reply by ghost** [reply]
well，i found the date of those all from 1/25/2050，you mean the wrong date lead to this problem？ since i had uploaded the photo of 2023，it could correctly display，however， why the mistake date would be hidden for some reason in immich？just curious about it.

**Reply by jrasm91** [reply]
Dates too far in the future break sometime else so we currently hide them.
